### ISSUE TITLE
ComfyUI isn't using my nvidia graphic card

### ISSUE THREAD

**Issue by Anouarsoufyani** [comfyanonymous/ComfyUI#863]
My PC configureation

CPU: AMD Ryzen 5 3600 6-Core
GPU: NVIDA GeForce GTX 1050 Ti
SSD: 512G

I lauched the webui using run_nvidia_gpu.bat but the nvidia card isn't working and the image does not appear.

![dzdzdzdzdzd](https://github.com/comfyanonymous/ComfyUI/assets/24847290/6a6caed3-b156-4a7e-8341-588fddad1a76)

**Comment by pennyvc** [comment]
Im faiiirly sure gtx 1050's can even run it... anyway, if it can actually run it.  check the cuda graph to the right of that, youll have to click on the little dropdown to see if its using the cuda core. (cuda is where all your AI stuff gets done.)

If this card is your second card and not the main one, you can force it to use a second card by entering this into the bat file that runs your comfyui  :  set CUDA_VISIBLE_DEVICES=1  put this before everything else in there.
this will only allow it to find the second gfx card basically.,
You should see an indication when launching it will say its using Cuda 1 rather than cuda 0  or something like that.

**Comment by Anouarsoufyani** [comment]
Alright, i didn't know that it couldn't work on my card, thanks for your reply.
unfortunately i can't even find cuda in the dropdown

![cuda](https://github.com/comfyanonymous/ComfyUI/assets/24847290/3393800f-0485-4ff1-b23d-fae9596586f6)

**Comment by Anouarsoufyani** [comment]
no, there is no errors showing up
with the run_nvidia_gpu.bat it just stops after the prompt is retrieved, and with the cpu one it works but it is very slow

**Comment by pennyvc** [comment]
Thats wierd, my 1050's have cuda.

**Comment by Anouarsoufyani** [comment]
yes all the drivers are up to date, maybe the 1050 ti doesn't have cuda

**Comment by Anouarsoufyani** [comment]
I updated the drivers yesterday and i still don't have cuda
![update drivers](https://github.com/comfyanonymous/ComfyUI/assets/24847290/a77f719f-1189-4517-b55d-7bb41f461410)

**Comment by ackzsel** [comment]
> I updated the drivers yesterday and i still don't have cuda. . .

Just checking: are you using drivers from the nvidia website, or do you use the drivers supplied by windows update?
